frontend/src/components/RoadmapOverview.tsx

```diff
@@ -31,20 +31,11 @@ export const RoadmapOverview = () => {
     return roadmapsVersions?.length || 0;
   };
 
-  const roadmapAverageRating = (dimension: TaskRatingDimension) => {
-    let sum = 0;
-    let count = 0;
-    roadmap!.tasks.forEach((task) => {
-      task.ratings.forEach((rating) => {
-        if (rating.dimension === dimension) {
-          sum += rating.value;
-          count += 1;
-        }
-      });
-    });
-
-    return count > 0 ? sum / count : 0;
-  };
+  const { value, work } = averageValueAndWork(roadmap?.tasks ?? []);
+  const numFormat = new Intl.NumberFormat(undefined, {
+    minimumFractionDigits: 0,
+    maximumFractionDigits: 2,
```

Comment:
The numFormat could be moved to /utils/string.ts (perhaps string.ts should be renamed to format.ts?)
```
export const numFormat = (
  minimumFractionDigits?: number,
  maximumFractionDigits?: number,
) =>
  new Intl.NumberFormat(undefined, {
    minimumFractionDigits,
    maximumFractionDigits,
  });
```

Reply:
I did consider that, but it would end up very thin wrapper around the constructor, with more limited options.

Such, specialised wrapper can be good, as it focuses the use case further, but I'm not sure if that is the case here.